portfolio/src/main/webapp/script.js

```diff
@@ -75,40 +75,48 @@ function addRandomBibleVerse() {
 }
 
 /**
- * Changes Navbar style depending on scroll position
+ * Changes Navbar style depending on scroll position.
  */
 window.addEventListener('scroll', function() {
   if (document.body.scrollTop > 80 || 
       document.documentElement.scrollTop > 80) {
     document.getElementById('navbar').style.padding = '30px 10px';
     document.getElementById('navbar').style.backgroundColor = 
         'rgba(0, 0, 0, 0.25)';
-    document.getElementById('logo').style.fontSize = '25px';
+    document.getElementById('myName').style.fontSize = '25px';
   }
   else {
     document.getElementById('navbar').style.padding = '80px 10px';
     document.getElementById('navbar').style.backgroundColor = 
         'rgba(0, 0, 0, 1)';
-    document.getElementById('logo').style.fontSize = '35px';
+    document.getElementById('myName').style.fontSize = '35px';
   }
 })
 
 /**
- * Fetches a message from the server and adds it to the DOM.
+ * Fetches all comments (submitted via the form) from the server
+ * and adds it to the DOM.
  */
-function getMessage() {
-  fetch('/data').then(response => response.json()).then((message) => {
-    const element = document.getElementById('message-container')
-    element.innerHTML = '';
-    element.appendChild(createListElement(message.string1));
-    element.appendChild(createListElement(message.string2));
-    element.appendChild(createListElement(message.string3));
+function getComment() {
+  let url = '/data?limit=' + document.getElementById("quantity").value.toString();
```

Comment:
Maybe use a more descriptive name than 'url' here?

Reply:
> Maybe use a more descriptive name than 'url' here?
Is fetchURL a good name?
> Also what happens if the value is empty, or not a number?
It is always a number equal or greater to zero (the form validates this).
> Even if your form element would not allow for such a thing, you should still be robust to that here in case 1) someone else wanted to leverage your servlet and didn't utilize data validation in their HTML, or 2) you migrated your own HTML and inadvertently allowed for those kinds of inputs.
How do I approach this? My idea is setting a default url, and if 'document.getElementById("quantity").value.toString()' gives a value >=0 then I append it to the string. Is that ok?

Reply:
I would say you can do two things: 1) if the data is invalid, bail and don't Post, or 2) fallback to a default URL, like you're saying. Up to you which you prefer! Probably longterm the best idea would be 1) + feedback to the user that their input was invalid, but that's going to be a decent amount of extra work I don't expect you to do for this project.

Reply:
Great, thanks for the tip! I went with number 2, so the user can at least have some sort of feedback when entering invalid values (the comments would get hidden), and I implemented this with try and catch.